app/admin/instances/[id]/page.tsx

```diff
@@ -219,8 +219,8 @@ export default async function InstanceDetailPage({
             instanceState={instanceState || ""}
           />
         ) : (
-          <div className="text-sm text-gray-500">
-            Browser Sessions: <span className="text-gray-400">N/A</span>
+          <div className="text-sm text-gray-700">
+            Browser Sessions: <span className="text-gray-600">N/A</span>
```

Comment:
![image](https://github.com/user-attachments/assets/851b75ce-9560-4ded-b861-dbc3676343d2)

I'm still seeing both issues. I think we need to update the BrowserSessionsIndicator above